service/src/main/java/bio/terra/workspace/service/workspace/GcpCloudContextService.java

```diff
@@ -30,12 +35,37 @@ public GcpCloudContextService(WorkspaceDao workspaceDao) {
    * @param workspaceId unique id of the workspace
    * @param cloudContext the GCP cloud context to create
    */
+  @Deprecated // TODO: PF-1238 remove
   public void createGcpCloudContext(
       UUID workspaceId, GcpCloudContext cloudContext, String flightId) {
     workspaceDao.createCloudContext(
         workspaceId, CloudPlatform.GCP, cloudContext.serialize(), flightId);
   }
 
+  /**
+   * Create the GCP cloud context for a workspace Supports {@link
+   * bio.terra.workspace.service.workspace.flight.CreateGcpContextFlightV2}
+   *
+   * @param workspaceId workspace id where the context is being created
+   * @param flightId flight doing the creating
+   */
+  public void createGcpCloudContextStart(UUID workspaceId, String flightId) {
+    workspaceDao.createCloudContextStart(workspaceId, CloudPlatform.GCP, flightId);
+  }
+
+  /**
+   * Update and unlock the GCP cloud context for a workspace. Store the cloud context data
+   *
+   * @param workspaceId workspace id of the context
+   * @param cloudContext cloud context data
+   * @param flightId flight completing the creation
+   */
+  public void createGcpCloudContextFinish(
```

Comment:
i noticed that the start and finish are called in separate step. what happens if one forget to call finish? Maybe add some documentation about the usage?